StRoot/StBFChain/StBFChain.cxx

```diff
@@ -1028,21 +1029,7 @@ Int_t StBFChain::Init() {
 	  if (GetOption("VmcGeo")) {
 	    path  = "./StarDb/VmcGeo:$STAR/StarDb/VmcGeo";
 	  }
-	  TString geom("Geometry.");
-	  geom +=  DbAlias[i].geometry;
-	  geom += ".C";
-	  Char_t *file = gSystem->Which(path.Data(),geom,kReadPermission);
-	  if (file) {
-	    LOG_INFO << "StBFChain::Init force load of " << file << endm;
-	    TString command = ".L "; command += file;
-	    gInterpreter->ProcessLine(command);
-	    gInterpreter->Calc("CreateTable()");
-	    command.ReplaceAll(".L ",".U ");
-	    gInterpreter->ProcessLine(command);
-	    delete [] file;
-	  } else {
-	    LOG_INFO << "StBFChain::Init file for geometry tag  " << geom << " has not been found in path" << path << endm;
-	  }
+	  CreateGeometry(DbAlias[i].geometry);
```

Comment:
CreateGeometry will not be found if the agml libraries are not loaded, e.g. if the AgML chain option is not specified.   We switched to using AgML for the geometry description ~ 2012.  Prior to that the geometry was supported from macros translated directly from the AgSTAR geometries (StarDb/AgiGeometry/).  Thpse macros also depend on "CreateGeometry"... albeit a much simpler macro.